projects/ion/src/lib/step/step.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { IonStepsComponent } from './step.component';
+import { IonModule } from '../ion.module';
+
+@NgModule({
+  declarations: [IonStepsComponent],
+  imports: [CommonModule, IonModule],
```

Comment:
```suggestion
  imports: [CommonModule, IonIconModule],
```
You're only using `ion-icon` on your component, so you only need to import IonIconModule here.